### PR TITLE
explicit docs for getter and setter, darken header, change last output, better feedback if you fail to load search index

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
Addresses https://github.com/dart-lang/dartdoc/issues/840

**Comment by sethladd** [comment]
@keertip

**Comment by sethladd** [comment]
Also addresses https://github.com/dart-lang/dartdoc/issues/854

**Comment by sethladd** [comment]
Also addresses https://github.com/dart-lang/dartdoc/issues/819

**Comment by sethladd** [comment]
Also addresses https://github.com/dart-lang/dartdoc/issues/811#issuecomment-134355317

**Comment by sethladd** [comment]
@devoncarew

File: bin/dartdoc.dart [hunk 1]
```diff
@@ -121,7 +121,7 @@ main(List<String> arguments) async {
 
   try {
     DartDocResults results = await dartdoc.generateDocs();
-    print('\nSuccess! Open file://${results.outDir.absolute.path}/index.html');
+    print('\nSuccess! Docs generated into ${results.outDir.absolute.path}');
```
Comment (devoncarew) [inline bin/dartdoc.dart]: +1

File: lib/src/model.dart [hunk 2]
```diff
@@ -1610,38 +1616,42 @@ class Accessor extends ModelElement implements EnclosedElement {
...
+  Library get library;
 
-  PropertyAccessorElement get getter;
-  PropertyAccessorElement get setter;
+  PropertyAccessorElement get _getter;
+  PropertyAccessorElement get _setter;
 
-  String computeDocumentationComment();
+  Accessor get getter {
+    if (_getter == null) return null;
```
Comment (devoncarew) [inline lib/src/model.dart]: could also do: `return _getter == null ? null : new ModelElement.from(_getter, library)`

**Comment by devoncarew** [comment]
lgtm!

File: lib/src/model.dart (hunk 2 above)
Comment (sethladd) [inline lib/src/model.dart]: Will do.

**Pull request closed without merging**